kibana-reports/server/routes/utils/__tests__/reportHelper.test.ts

```diff
@@ -14,43 +14,48 @@
  */
 
 import 'regenerator-runtime/runtime';
-import { generatePDF, generatePNG } from '../visualReportHelper';
+import { createVisualReport } from '../reportHelper';
 
-describe('test generate report', () => {
-  test('generate PNG successfully', async () => {
-    expect.assertions(1);
+describe('test create visual report', () => {
+  test('create png report', async () => {
+    expect.assertions(3);
     const input = {
-      url: 'https://opendistro.github.io/for-elasticsearch-docs/',
-      itemName: 'test',
-      windowWidth: 1200,
-      windowLength: 800,
+      report_name: 'Zhongnan_daily_report/4pm',
+      report_source: 'Dashboard',
+      report_type: 'Download',
+      description: 'Hi this is your dashboard',
+      report_params: {
+        url: 'https://opendistro.github.io/for-elasticsearch-docs/',
```

Comment:
Can we change this to the demo elastic link? After the trim report is merged in, this test case will not pass if the url is not a Kibana dashboard

Reply:
changed